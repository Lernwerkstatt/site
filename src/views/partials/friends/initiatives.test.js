const initiative = require("./initiatives.hbs");

describe("initiatives.hbs", () => {
  it("should match the snapshot", () => {
    expect(
      initiative({
        name: "Initiative",
        link: "http://www.die-lernwerkstatt.org/",
        text: "This is an initiative"
      })
    ).toMatchSnapshot();
  });
});
