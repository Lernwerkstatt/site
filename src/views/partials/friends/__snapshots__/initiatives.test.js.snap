// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`initiatives.hbs should match the snapshot 1`] = `
"<div class=\\"card\\">
  <div class=\\"card-body\\">
    <a href=\\"http://www.die-lernwerkstatt.org/\\" target=\\"_blank\\">
      <h5 class=\\"card-title\\">Initiative</h5>
    </a>
    <p class=\\"card-text\\">This is an initiative</p>
  </div>
</div>
"
`;
